### PR TITLE
Fix wrong npm version introduced with d202b07

### PR DESCRIPTION
**Pull request by MariusRumpf**
Commit d202b07 introduced a not existing npm version of >= 5 which does not exist. The correct npm version is checked during installation which might otherwise lead to an error. Conclusion might be that this was meant to be node >= 5 like mentioned in #1197.

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -7,7 +7,7 @@
     "url": "git://github.com/mxstbr/react-boilerplate.git"
   },
   "engines": {
-    "npm": ">=5"
+    "npm": ">=3"
```
Comment (samit4me) [inline package.json]: Should we also add `"node": ">=5"`? Just so it's clearer?

Comment (gihrig) [inline package.json]: Agreed, but should be "node": ">=4". Reason: if Node 4 is to be deprecated other areas need to be adjusted as well (travis and appveyor configs, docs, other?). These changes are out of scope for this PR. I guess it could be argued that `engines: node` could even be out of scope.

Comment (samit4me) [inline package.json]: This was done as part of #1197, so we should be good!

Comment (gihrig) [inline package.json]: @samit4me You're right. My error 😊
